Cargo.toml

```diff
@@ -23,18 +23,23 @@ lazy_static = "1.0"
 memmap2 = "0.5.0"
 log = "0.4"
 thiserror = "1.0.30"
-wayland-backend = "=0.1.0-beta.8"
-wayland-client = "=0.30.0-beta.8"
-wayland-protocols = { version = "=0.30.0-beta.8", features = ["client", "unstable"] }
-wayland-protocols-wlr = { version = "=0.1.0-beta.8", features = ["client"] }
-wayland-cursor = "=0.30.0-beta.8"
-
+# wayland-backend = "=0.1.0-beta.8"
+# wayland-client = "=0.30.0-beta.8"
+# wayland-protocols = { version = "=0.30.0-beta.8", features = ["client", "unstable"] }
+# wayland-protocols-wlr = { version = "=0.1.0-beta.8", features = ["client"] }
+# wayland-cursor = "=0.30.0-beta.8"
+wayland-backend = { git="https://github.com/smithay/wayland-rs" }
+wayland-client = { git="https://github.com/smithay/wayland-rs" }
+wayland-protocols = { git="https://github.com/smithay/wayland-rs", features = ["client", "unstable"] }
+wayland-protocols-wlr = { git="https://github.com/smithay/wayland-rs", features = ["client"] }
+wayland-cursor = { git="https://github.com/smithay/wayland-rs" }
 # Explicit dependency until release
 xkbcommon = { git = "https://github.com/rust-x-bindings/xkbcommon-rs", optional = true, features = ["wayland"] }
 calloop = { version = "0.10.0", optional = true }
 
 [features]
-default = ["calloop", "xkbcommon"]
+default = ["calloop", "xkbcommon", "system"]
+system = ["wayland-backend/client_system"]
```

Comment:
Why add this feature, and why make it default?

Also this looks unrelated to this PR and should be discussed in a separate issue/PR imo.

Reply:
I've been using as_ptr() for a while and recently needed to add this feature to make the method available again. I figured all features were enabled by default, so I enabled it by default. I can move it to a different PR though.

Reply:
Mostly the encouraged method to use `as_ptr()` for a project is by itself depending directly on `wayland-backend` to activate the feature, as this method is part of `wayland-backend` API. So I don't see any reason to add that feature to SCTK, given it has 0 impact on the crate.

Reply:
ok that change has been removed